### DISCUSSION TITLE
[Feature Request] Site Group as a Field Condition in Field Layouts

### DISCUSSION THREAD

**Discussion by mike-moreau** [craftcms/cms#15625] [Ideas] [answered]
It would be convenient to have a condition for "Site Group" to display a custom field.

For cases when there are 3-4 sites in a group and the possibility of more to be added later, this would prevent me from having to edit conditionals after a new site is added to the group.

<img width="924" alt="Screenshot 2024-08-29 at 8 42 33 AM" src="https://github.com/user-attachments/assets/40b5c569-2071-4746-91e2-073ca0582fdf">

**Comment by mmikkel** [comment]
This would be super useful.

**Comment by brandonkelly** [comment, marked as answer]
Easy enough. Added for 4.12 and 5.4 (0ed996866d7b03e25f08bb5e088aa5204d37bf13).

**Reply by brandonkelly** [reply]
Craft 4.12.0 and 5.4.0 are out now with the Site Group condition rule.
